kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveKeyValueStore.java

```diff
@@ -232,8 +219,4 @@ public KeyValueIterator<Bytes, byte[]> reverseAll() {
     return operations.reverseAll();
   }
 
-  // Visible for testing
-  KeyValueOperations getOperations() {
```

Comment:
Didn't seem to be used (we can of course add it back if any test actually does need it, but I don't think we need this AND the `kvOperationsProvider`)

Reply:
ah yes, I had written a test to make sure it creates the right operations (global vs local) but then realized the test was kind-of nonsense sine it mocked so much. I deleted it and forgot to update this